scripts/js/commands/api/syncDevDocs.ts

```diff
@@ -47,10 +47,10 @@ zxMain(async () => {
 
   await updateHtmlArtifacts({ qiskitUrl, runtimeUrl });
 
-  if (!qiskitIsRc) {
+  if (qiskitUrl) {
```

Comment:
This change has no impact because `qiskitUrl` is only set if `!qiskitIsRc`. I did it because 
a) I think it's clearer code, and 
b) I was manually commenting out the code that sets `qiskitUrl` and was annoyed that this conditional still was executing.